### DISCUSSION TITLE
Homeassisant

### DISCUSSION THREAD

**Discussion by vgr28** [FI18-Trainees/vvspy#81] [Q&A]
Hallo, gibt es eine Intergrationsmöglichkeit in Homeassisant? Falls ja, würde ich mich über einen Support oder Beschreibung freuen..

V.G.

**Comment by zaanposni** [comment]
Nein, momentan nicht. vvspy bietet lediglich die Möglichkeit in Python mit der API zu interagieren.
Vielleicht hilft auch [vvs_direct_connect](https://github.com/aschuma/vvs_direct_connect) von aschuma.

Falls du etwas eigenes für Homeassistant programmierst, würde ich mich freuen, wenn du es auch öffentlich teilst, falls es noch jemand anderes braucht :)
